client-ts/Microsoft.AspNetCore.SignalR.Client.TS/Transports.ts

```diff
@@ -195,6 +195,7 @@ export class LongPollingTransport implements ITransport {
         }
 
         let pollXhr = new XMLHttpRequest();
+        pollXhr.setRequestHeader("User-Agent", "Microsoft.AspNetCore.SignalR.Client/1.0.0-alpha");
```

Comment:
Move this to a separate file so that we don't have the same string in two places because the will go out of sync (also makes #763 easier)

Reply:
Also, I think the value should be the same as for the header from the C# client - maybe with `-js` so we can tell from the logs what kind of client it was.